test/v3/index.test.js

```diff
@@ -71,6 +71,17 @@ describe("cli", () => {
       const expected = eol.lf(fs.readFileSync(new URL(`./expected/${filename}`, cwd), "utf8"));
       expect(generated).to.equal(expected);
     });
+
+    it(`reads ${schema} spec (v3) from file (alphabetize)`, async () => {
```

Comment:
These CLI tests are “snapshots,” where the output is so long and complicated it’s impossible to read through everything and see if it’s working as expected. But that’s their design—they’re mostly “smoke tests” that test that complex behavior can still work given a complex, real-world schema.

Alphabetization shouldn’t live here IMO, because it’s unclear if the original schemas were even unsorted in the first place (some examples are!).

Instead, I’d rather see the alphabetization tests live the spec tests themselves, testing that for every major type of OpenAPI object (`paths`, `components`, `operations`), and even adding tests for specific types (do object `properties` keys get sorted? Do endpoints? Do content types? `enum`s? etc. Because I think it’s very likely that some structures may be unsorted yet, and digging further into each data type would assure that.